rust-toolchain.toml

```diff
@@ -0,0 +1,3 @@
+[toolchain]
+channel = "nightly-2022-09-08"
```

Comment:
I don't find documents about MSRV so just assume that we use the nightly version as usual.

Add this file as it's required to enable feature gates (`drain_filter` in this patch). Although it's possible to write a utility function, if we can just use stdlib, I'd prefer to use it.

Reply:
To me, the toolchain have to be stable, if it is not set. I am strongly discourage nightly requirement on production driver.